pkgs/development/python-modules/google-crc32c/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, cffi }:
```

Comment:
Passing in the whole of `pkgs` as an argument makes it really hard to keep track of what dependencies are related to this package and makes it quite painful to override specific arguments.